spec/features/uva_work_form_spec.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require HykuAddons::Engine.root.join("spec", "support", "fill_in_fields.rb").to_s
+require HykuAddons::Engine.root.join("spec", "support", "work_form_helpers.rb").to_s
+
+RSpec.feature "Create a UvaWork", js: true do
+  let(:work_type) { "uva_work" }
+
+  let(:model) { work_type.classify.constantize }
+  let(:field_config) { "hyrax/#{work_type}_form".classify.constantize.field_configs }
+  let(:user) { User.new(email: "test@example.com") { |u| u.save(validate: false) } }
+  let(:admin_set_id) { AdminSet.find_or_create_default_admin_set_id }
+  let(:permission_template) { Hyrax::PermissionTemplate.find_or_create_by!(source_id: admin_set_id) }
+  let(:workflow) { Sipity::Workflow.create!(active: true, name: "test-workflow", permission_template: permission_template) }
+  let(:new_work_path) { "concern/#{work_type.to_s.pluralize}/new" }
+  let(:permission_options) do
+    { permission_template_id: permission_template.id, agent_type: "user", agent_id: user.user_key, access: "deposit" }
+  end
+  # The organisation option changes depending on the local, so we need to use this to ensure we select the right one
+  let(:organisation_option) { HykuAddons::NameTypeService.new(model: model).active_elements.last }
+
+  let(:title) { "UVA Work Item" }
+  let(:creator) do
```

Comment:
Both this an the `ubiquity_template_work_form_spec` share a lot of definitions and expectations.  Is there any sensible way to bundle these up into groups and share them?